client/components/select-dropdown/separator.jsx

```diff
@@ -3,15 +3,10 @@
 /**
  * External Dependencies
  */
-var React = require( 'react' );
+import React from 'react';
 
-var SelectDropdownSeparator = React.createClass( {
-
-	render: function() {
-		return (
-			<li className="select-dropdown__separator" />
-		);
-	}
-} );
-
-module.exports = SelectDropdownSeparator;
+export default function SelectDropdownSeparator() {
```

Comment:
As far as I know I am already using a pure function here.
From the snippet of code I guess you're suggesting to use an arrow function, in place of a classic function expression; am I right?

Something like:

```js
export default () => {
  return ...
}
```

In this case, since arrow function are anonymous by default, we lose the component nice name.
There's a way to restore the name:

```js
const SelectDropdownSeparator = () => {
  return ...
};

export default SelectDropdownSeparator;
```

But it is much verbose.

Please let me know what you think about this. I'll stick with classic function expression for the moment.